includes/admin/upgrades/upgrade-functions.php

```diff
@@ -156,6 +156,10 @@ function give_do_automatic_upgrades() {
 			give_v270_upgrades();
 
 			$did_upgrade = true;
+
+		case version_compare( $give_version, '2.9.0', '<' ):
+			give_v290_remove_old_export_files();
+			$did_upgrade = true;
```

Comment:
@ravinderk Am I adding this automatic upgrade correctly? I want this to run once someone upgrades to 2.9.0.

Reply:
This is right way.